File: ARCADE/The Core/Level 2_4: isInfiniteProcess?.cs
    using System;
     
    public class Test
    {
    	public static bool isInfiniteProcess(int a, int b){
            if (b >= a && ((b - a) % 2 == 0)) return false;
            return true;
        }
    	public static void Main()
    	{
    		int a = 2, b = 6;
    		Console.WriteLine(isInfiniteProcess(a, b));
    		int c = 2, d = 3;
    		Console.WriteLine(isInfiniteProcess(c, d));
    	}
    }

File: ARCADE/The Core/Level 2_4: isInfiniteProcess?.pas
program isInfiniteProcess;

function isInfiniteProcess(a: integer; b: integer): boolean;
begin
     if((b >= a) and ((b - a) mod 2 = 0)) then isInfiniteProcess := false
     else isInfiniteProcess := true;
end;

var a, b : integer;
begin
	a := 2; b := 6;
	writeln(isInfiniteProcess(a, b));
	a := 2; b := 3;
	writeln(isInfiniteProcess(a, b));
end.
